p4rt_app/sonic/app_db_manager.cc

```diff
@@ -400,34 +384,6 @@ absl::Status AppendExtTableDefinition(
   return absl::OkStatus();
 }
 
-// Publish set of tables in json formatted string to AppDb
-absl::StatusOr<std::string> PublishExtTablesDefinitionToAppDb(
```

Comment:
Ext Table code applies to sonic-pins, pls restore this code back.
@rkavitha-hcl

Reply:
Done